modules/vocabularies/src/main/java/ca/sickkids/ccm/lfs/vocabularies/internal/OntologyIndexerUtils.java

```diff
@@ -46,6 +48,9 @@
     /** The list which holds all JCR vocabulary nodes associated with a vocabulary to be checked-in. */
     private static final ThreadLocal<List<Node>> NODES_TO_CHECK_IN = ThreadLocal.withInitial(ArrayList::new);
 
+    /** The list which holds all root vocabulary nodes to add to vocabulary node. */
```

Comment:
This sounds a bit awkward.

```suggestion
    /** The list which holds all root terms. */
```